core/src/main/java/com/ibm/wala/ipa/callgraph/AnalysisScope.java

```diff
@@ -345,6 +351,30 @@ public String toString() {
     return result.toString();
   }
 
+  public String toJson() {
+    HashMap<String, Object> res = new HashMap<>();
+    HashMap<String, ArrayList<String>> loaders = new HashMap<>();
+    for (ClassLoaderReference loader : loadersByName.values()) {
+      ArrayList<String> arr = new ArrayList<>();
+      for (Module m : getModules(loader)) {
+        arr.add(m.toString());
+      }
```

Comment:
Here I think you could use the `Iterator2Collection.toList` type to avoid writing this loop: https://github.com/wala/WALA/blob/675cee88cd6fbfe82e7ad20f5d98e97430acb574/util/src/main/java/com/ibm/wala/util/collections/Iterator2Collection.java#L39

Reply:
This comment is still not addressed

Reply:
We decided to just punt on this, as it was causing other issues